acceptance/gossip_peerings_test.go

```diff
@@ -45,11 +44,6 @@ func checkGossip(t *testing.T, l *localcluster.Cluster, d time.Duration,
 		case <-stopper:
 			t.Fatalf("interrupted")
 			return nil
-		case e := <-l.Events:
-			if log.V(1) {
-				log.Infof("%+v", e)
-			}
-			return fmt.Errorf("event: %+v", e)
```

Comment:
but all tests should by default make sure that they don't receive unexpected events. The only other alternative is not using `cluster.AssertAndStop(...)` but having a wrapper around `.Stop()` which lives in the acceptance tests. But then it's not obvious that in a test you're supposed to use that wrapper instead of verbatim `Stop()`. That was my motivation for `AssertAndStop`, anyway.

On a more general level, I'm not sure I want tests to even construct the expected events manually. After all, if a test calls `Nodes[3].Restart()` it should be understood that there's a restart event for that node. A logical next step would be adding the expected events to the cluster state and asserting on shutdown, which suggests that `AssertAndStop(...)` is a good middle ground for now.